### PR TITLE
Chore/e2e purchase licenses estimation

### PR DESCRIPTION
**Pull request by santiagonoguez**
## Description
E2E for add display licenses form and estimation. E2E for subscription update still not implemented.

[stage-9]

## Motivation and Context
E2E for epic

## How Has This Been Tested?
These are tests. No runtime changes ( just 2 ids and a class added to HTML file ).

## Release Plan:
To be merged to feature/purchase-licenses

- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
-

### CODE REVIEW

**Comment by santiagonoguez** [comment]
@ezequielc @alex-deaconu don't forget to review this, thanks !

File: test/e2e/registration/cases/subscription-edit.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        });
+
+        it("should display coupon code field - again", function() {
+          addDisplayLicensesPage.getAddCouponCodeLink().click();
+
+          expect(addDisplayLicensesPage.getCouponCodeInput().isDisplayed()).to.eventually.be.true;
+        });
+
+        it("should apply a coupon", function() {
+          addDisplayLicensesPage.getCouponCodeInput().sendKeys('save50');
```
Comment (alex-deaconu) [inline test/e2e/registration/cases/subscription-edit.js:147]: @santiagonoguez I would not test coupons, as they could change at any point and break this. Thanks.

Comment (santiagonoguez) [reply test/e2e/registration/cases/subscription-edit.js:147]: @alex-deaconu sounds good, I'll remove those steps then, thanks !

File: test/e2e/registration/cases/subscription-edit.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        });
+
+        it("should show Add Coupon Code link", function() {
+          expect(addDisplayLicensesPage.getAddCouponCodeLink().isDisplayed()).to.eventually.be.true;
+        });
+      });
+
+      describe("update display licenses: ", function() {
+        before(function() {
+          browser.sleep(2000); // give some time to actual totals to be displayed. Otherwise, just '$' is detected.
```
Comment (alex-deaconu) [inline test/e2e/registration/cases/subscription-edit.js:71]: @santiagonoguez to confirm, this is not related to the spinner load just a UI update? Thanks.

Comment (santiagonoguez) [reply test/e2e/registration/cases/subscription-edit.js:71]: @alex-deaconu yes, even if spinner has finished, sometimes it gets the amount text as '$' if I don't add this wait. 

Do you think is worth to test if that doesn't happen on CI ? Maybe is just my local env only.

Comment (alex-deaconu) [reply test/e2e/registration/cases/subscription-edit.js:71]: @santiagonoguez I'm just confirming this is not a wait for spinner, which it's not. Wondering why there is latency in this case, but not a big deal. Let's leave as is. Thanks.

File: test/e2e/registration/cases/subscription-edit.js (hunk 1 above)
Comment (santiagonoguez) [reply test/e2e/registration/cases/subscription-edit.js]: @alex-deaconu I've removed it, thanks !

**Review by alex-deaconu — APPROVED** [review]
@santiagonoguez looks good, thanks for the updates 👍
